src/proposals/20250417/SparkEthereum_20250417.t.sol

```diff
@@ -0,0 +1,190 @@
+// SPDX-License-Identifier: AGPL-3.0
+pragma solidity ^0.8.10;
+
+import 'src/test-harness/SparkTestBase.sol';
+
+import { IERC20 }   from 'forge-std/interfaces/IERC20.sol';
```

Comment:
Many imports aren't used. Will review the list after making sure tests are sufficient

Reply:
removed